### PR TITLE
Update instructions for correct versions of go/node brew formulas

### PR DESCRIPTION
**Pull request by reggieriser**
## Description

It looks like our docs/scripts suggest installing the latest versions of `go` and `node` via homebrew.  As of this writing, that will get you go 1.13.x and node 12.x, but the project is currently on go 1.12.x and node 10.x.  This PR adjusts the instructions to use the version-specific brew formulas for `go@1.12` and `node@10`.

## Code Review Verification Steps

* [ ] Request review from a member of a different team.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,10 +76,9 @@ There are a number of things you'll need at a minimum to be able to check out, d
 
 * Install [Homebrew](https://brew.sh)
   * Use the following command `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
-* We always use the latest version of Go unless there's a known conflict (which will be announced by the team).
-  * Install it with Homebrew: `brew install go`
-  * Pin it, so that you don't accidentally upgrade before we upgrade the project: `brew pin go`
-  * When we upgrade the project's go version, unpin, upgrade, and then re-pin: `brew unpin go; brew upgrade go; brew pin go`
+* We normally use the latest version of Go unless there's a known conflict (which will be announced by the team) or if we're in the time period just after a new version has been released.
+  * We are still using go 1.12 for now; we will be migrating to go 1.13 soon.
```
Comment (chrisgilmerproj) [inline README.md]: Maybe mention what triggers an update? I think that its generally an update to the circleci docker container.

Comment (reggieriser) [reply README.md]: Added a note here about the trigger.

File: scripts/prereqs [hunk 2]
```diff
@@ -15,15 +15,15 @@ function has() {
...
+has go "brew install go@1.12 && brew link --force go@1.12"
 has bash "brew install bash"
 has yarn "brew install yarn"
 has pre-commit "brew install pre-commit"
 has shellcheck "brew install shellcheck"
 has docker "Get Docker CE for Mac from https://download.docker.com/mac/stable/Docker.dmg"
 has psql "brew install postgresql"
 has go-bindata "brew install go-bindata"
-has node "brew install node"
+has node "brew install node@10 && brew link --force node@10"
```
Comment (chrisgilmerproj) [inline scripts/prereqs]: I had to run `brew link --overwrite node@10 --force` on this

Comment (reggieriser) [reply scripts/prereqs]: Hmm, wouldn't that imply you already had a version of `node` linked if you had to overwrite it?  And if you had `node` linked, not sure why these instructions would display.  At any rate, I'd prefer the user make a conscious choice to overwrite if necessary (I think brew prompts you about this).

Comment (chrisgilmerproj) [reply scripts/prereqs]: I'm with you on that. I am thinking of the case of migrating folks who already have node installed I guess. Let's move forward with your change and we'll see what happens!

**Pull request merged**